lib/Backend/GlobOptBailOut.cpp

```diff
@@ -916,7 +916,7 @@ GlobOpt::MayNeedBailOnImplicitCall(IR::Opnd * opnd, Value *val, bool callsToPrim
     case IR::OpndKindReg:
         // Only need implicit call if the operation will call ToPrimitive and we haven't prove
         // that it is already a primitive
-        return callsToPrimitive &&
+        return //callsToPrimitive &&
```

Comment:
Oh, yeah. Thanks.

--Paul

From: Curtis Man [mailto:notifications@github.com]
Sent: Thursday, June 23, 2016 10:15 AM
To: Microsoft/ChakraCore <ChakraCore@noreply.github.com>
Cc: Paul Leathers <pleath@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Microsoft/ChakraCore] Restructure OpHasImplicitCall/OpCallsValueOf (#1162)


In lib/Backend/GlobOptBailOut.cpp<https://github.com/Microsoft/ChakraCore/pull/1162#discussion_r68275422>:

> @@ -916,7 +916,7 @@ GlobOpt::MayNeedBailOnImplicitCall(IR::Opnd * opnd, Value *val, bool callsToPrim

>      case IR::OpndKindReg:

>          // Only need implicit call if the operation will call ToPrimitive and we haven't prove

>          // that it is already a primitive

> -        return callsToPrimitive &&

> +        return //callsToPrimitive &&

Remove commented code?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/ChakraCore/pull/1162/files/399fcf51c410a7ab619374ffc00f5b0ba4bbf54d#r68275422>, or mute the thread<https://github.com/notifications/unsubscribe/APF8RNPtXMVYbQhunnlhuc76A2bPUB2_ks5qOr8fgaJpZM4I7C5E>.